pkgs/development/libraries/libeb/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, perl, zlib }:
+stdenv.mkDerivation rec {
+  name = "libeb-${version}";
+  version = "4.4.3";
+
+  src = fetchurl {
+    url = "ftp://ftp.sra.co.jp/pub/misc/eb/eb-${version}.tar.bz2";
+    sha256 = "0psbdzirazfnn02hp3gsx7xxss9f1brv4ywp6a15ihvggjki1rxb";
+  };
+
+  nativeBuildInputs = [ perl ];
+  buildInputs = [ zlib ];
+
+  meta = with stdenv.lib; {
+    description = "C library for accessing CD-ROM books, supports EB, EBG, EBXA, EBXA-C, S-EBXA and EPWING formats.";
```

Comment:
Please remove dot at the end. And make description little bit shorter.
For long description you can use `longDescription`.